style.css

```diff
@@ -1 +1,172 @@
 /*  Add your styling here */
+body {
+  font: 1em sans-serif;
+  align-content: center;
+  align-items: center;
+}
+
+header {
+  width: 95%;
+  margin-left: auto;
+  margin-right: auto;
+  height: auto;
+  padding-top: 2%;
+  padding-bottom: 2%;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  background-color: rgb(253, 236, 239);
+
+  border: 3px darkblue solid;
+}
+
+header img {
+  width: 50px;
+  height: 50px;
+
+  border: 1px red solid;
+}
+
+header div {
+  width: 60%;
+  display: table;
+
+  border: 1px red solid;
+}
+
+header div h4 {
+  width: auto;
+  text-align: right;
+  opacity: 0.6;
+  font-style: italic;
+  margin: 0;
+
+  border: 1px red solid;
+}
+header nav {
+  text-align: right;
+
+  border: 1px red solid;
+}
+
+#welcome {
```

Comment:
Don't use IDs in your CSS. You can use IDs for JavaScript, but when adding styles to HTML, target your elements using classes.